aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Merger.java

```diff
@@ -77,22 +76,24 @@
             // Update the metadata for it
             .andThen(save(metadata))
             // Let the world know that we've done a good thing.
-            .andThen(announceSuccessfulMerge(modelWithMetadata));
+            .doOnComplete(() -> {
+                announceSuccessfulMerge(modelWithMetadata);
+                LOG.verbose("Remote model update was sync'd down into local storage: " + modelWithMetadata);
+            })
+            .doOnError(failure ->
+                LOG.warn("Failed to sync remote model into local storage: " + modelWithMetadata)
```

Comment:
Can some details about the error also be logged?

Reply:
Whoops, appended `, failure` into the log statement.